src/template/render.rs

```diff
@@ -471,9 +473,45 @@ impl<'a> ChainTemplateSource<'a> {
         };
         debug_span!("Executing command", ?command)
             .in_scope(|| async {
-                let output = Command::new(program)
+                // Spawn the command process
+                let mut process = Command::new(program)
                     .args(args)
-                    .output()
+                    .stdin(Stdio::piped())
+                    .stdout(Stdio::piped())
+                    .stderr(Stdio::piped())
```

Comment:
We're not using stderr, so might as well dump it and save on memory
```suggestion
                    .stderr(Stdio::null())
```

Reply:
It is debug logged.

```rust
debug!(
    stdout = %String::from_utf8_lossy(&output.stdout),
    stderr = %String::from_utf8_lossy(&output.stderr),
    "Command success"
);
```

I found it useful. We can also enhance it with exit code.

It is useless if we don't log. Maybe we can check if it is debug log level but I don't think it is worth bothering.

Reply:
Ah forgot about that, thanks. It's fine like it is then.